torch/jit/_recursive.py

```diff
@@ -65,6 +65,19 @@ def make_stubs_from_exported_methods(mod):
 
     return stubs
 
+def is_jit_ignored_property(module, prop):
+    user_annotated_ignored_attributes = getattr(module, "__jit_ignored_attributes__", list())
+
+    def get_properties_names(module):
+        return set(k for k, v in vars(module).items() if isinstance(v, property))
+
+    properties = get_properties_names(type(module))
+    user_annoted_ignored_properties = set()
+
+    for ignored_attr in user_annotated_ignored_attributes:
+        if ignored_attr in properties:
+            user_annoted_ignored_properties.add(ignored_attr)
```

Comment:
IMHO, it might be better to revise this method to return the set of `user_annotated_ignored_properties` since the caller usually checks every attribute with this method, that results in O(N^2) complexity where N is number of properties. If you return the set and have caller query into the set, then complexity is O(N)

Reply:
Hmm. I think I am returning `user_annotated_ignored_properties` as a set. Is that what you mean?

Reply:
Or do you mean `user_annotated_ignored_attributes` should be a set?

Reply:
I mean this entire method should not be checking if an individual property is an ignored property or not, instead it should return the set directly so when caller makes repeated inquiries of all attributes on a module, it is cheaper. (ie, the set is only constructed once rather than N times)

Reply:
oh yeah makes sense. will update.